### ISSUE TITLE
Jetengine Repeater Native dynamic accordion

### ISSUE THREAD

**Issue by nreljed** [CrocoBlock/suggestions#6454]
How to use just jetengine and dynamic repeater widget to simulate an accordian without going through this very [heavy method](https://crocoblock.com/troubleshooting/articles/display-custom-repeater-fields-in-accordion/) of jettab + dev plugin?
The idea is to make things simpler 
I don't know what elementor's nested accordion will offer in the coming months but I don't think it will be dynamic.
I propose to evolve repeater widget to simulate an Accordion

The questions must remain in the target post
![Capture d’écran 2023-02-01 103555](https://user-images.githubusercontent.com/82481592/216005839-b49a80d8-c6f1-45c9-b852-d542581722c4.png)

Everything is there, an enable accordion option is missing, this should be totally possible with the least effort.
![Capture d’écran 2023-02-01 103826](https://user-images.githubusercontent.com/82481592/216006396-0651b990-da94-4bd6-8e9a-24a817c8c301.png)

what jetengine and users earn ?
a huge gain in simplicity and open another door to more possibilities and flexibility, far from the traditional method of adding more plugins ,more code,  and more conflicts to manage for the future !
everyone wins 100% with this method

**Comment by SolunaCG** [comment]
I fully agree! I´d love to see this implemented, too. I had similar problems with my last project where I had to go through the whole heavy method that @nreljed shared. An easier way would be rlly appreciated!

**Comment by MjHead** [comment]
Since 3.1.0 we added ability to use Query on widgets like accordion. Also we added Repeater Query type to JetEngine - you can use this combination. At the moment compatibility added to our widgets where repeater control is used to add content, like JetTabs and part of widgets from JetElements. https://github.com/MjHead/jet-engine-custom-query-gateway - here is an example how the same functionality could be implemented with some of native Elementor widgets. The only problem - it requires couple of lines of code which will be added with JetEngine. You can wait until this release if you want or replace it by yourself. Go to `includes/components/query-builder/query-gateway/manager.php` file and replace there `$widget->get_settings_for_display` with `$widget->get_settings`

**Comment by nreljed** [comment]
@MjHead thank you for the reply i will wait for a next update 🤞

**Comment by tizum** [comment]
That's awesome, could you add support for Options pages as well?

Will it be possible to render the media images?

https://github.com/Crocoblock/suggestions/issues/6463

**Comment by aalexandr21** [comment]
was released in 3.1.0
